AnabiControllers.Tests/AssetStorageSpaceControllerTest.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Anabi;
+using Anabi.DataAccess.Ef;
+using Anabi.Domain.Asset.Commands;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Security.Principal;
+using System.Threading.Tasks;
+using Anabi.Domain;
+using System.Threading;
+using Anabi.Domain.Asset;
+
+namespace AnabiControllers.Tests
+{
+    [TestClass]
+    public class AssetStorageSpaceControllerTest
+    {
+        private AnabiContext context;
+        private IMapper mapper;
+        private IPrincipal principal;
+
+        private BaseHandlerNeeds BasicNeeds => new BaseHandlerNeeds(context, mapper, principal);
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            Setup();
+        }
+
+        [TestCleanup]
+        public void CleanUp()
+        {
+            context.Dispose();
+            context = null;
+
+            mapper = null;
+            Mapper.Reset();
+        }
+
+        [TestMethod]
+        public async Task AddAssetStorageSpace()
+        {
+            var queryHandler = new AssetStorageSpaceHandler(BasicNeeds);
+            var entity = new AddAssetStorageSpace
+            {
+                AssetId = 1002,
+                EntryDate = DateTime.Now,
+                StorageSpaceId = 1
+            };
+
+            await queryHandler.Handle(entity, CancellationToken.None);
+
+            var cat = await context.AssetStorageSpaces.FirstOrDefaultAsync(a => a.Id == 1);
```

Comment:
Testul poate esua in anumite situatii pt ca ii specifici Id-ul din baza de date. Poti cauta baza de date (AssetStorageSpaces) dupa AssetId si StorageSpaceId, dar nu dupa id-ul entitatii din baza de date. Id-ul este generat automat si s-ar putea sa nu fie intotdeauna 1.